habitat-baselines/habitat_baselines/config/habitat_baselines/rl/agent/pop_play.yaml

```diff
@@ -1,3 +1,4 @@
 type: "MultiAgentAccessMgr"
-num_total_agents: 2
-num_active_agents: 2
+num_agent_types: 2
```

Comment:
I believe it is population. This yaml indicates how agents should be sampled during training, based on the agents we defined. Previously we had `num_total_agents` out of which we sampled `num_active_agents`. This means sometimes we would sample two times agent 0, sometimes two times agent 1 etc.

The change is made so that we specify how many agents we should sample from type0, how many form type 1 etc, so that we can always make sure we have 1 human and 1 robot during training for instance.